app/components/document/tracking_bookmark_component.html.erb

```diff
@@ -12,7 +12,6 @@
     <label class="toggle-bookmark-label" data-checkboxsubmit-target="label">
       <input type="checkbox" class="toggle-bookmark-input <%= bookmark_icon ? 'd-none' : '' %>" data-action="click->analytics#trackBookmark" data-checkboxsubmit-target="checkbox" <%= 'checked="checked"' if bookmarked? %>>
       <%= bookmark_icon %>
-      <span data-checkboxsubmit-target="span"><%= bookmarked? ? t('blacklight.search.bookmarks.present') : t('blacklight.search.bookmarks.absent') %></span>
```

Comment:
This causes a javascript error:
<img width="361" alt="Screenshot 2025-06-26 at 11 14 45 AM" src="https://github.com/user-attachments/assets/abd9b1f1-8524-4ef1-8176-4a84ee5e9b0e" />